package.json

```diff
@@ -71,5 +75,21 @@
         "protobufjs": "^7.2.4",
         "semver": "^7.5.3",
         "nx": "16.5.5"
-    }
+    },
+    "lint-staged": {
+        "packages/**/src/**/*.ts": [
+          "prettier --write"
+        ],
+        "!(packages/**/src/dist/)*.ts": [
```

Comment:
We don't check dist in, is this just a nice to have for local troubleshooting view?

Reply:
Yes, I think we don't need to check and format anything under dist folder. Also do you think we should also include tests to be properly formated upon changes and commits ? 
```
        "packages/**/tests/**/*.ts": [
          "prettier --write"
        ],
```

Reply:
Yeah we should format test files also

Reply:
Added them as well.